<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1261</width>
    <height>773</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="main_frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>-30</y>
      <width>1201</width>
      <height>841</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(62, 62, 62);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="widget_frame">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>230</y>
       <width>371</width>
       <height>231</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(104, 104, 104)</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="timer_label">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>10</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 18pt &quot;Verdana&quot;;
color:#ffca18;
</string>
      </property>
      <property name="text">
       <string>screen record frame</string>
      </property>
     </widget>
     <widget class="QPushButton" name="start_record_button">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#start_record_button{
background:#FFF;
border:none;
border-radius:10px;
}


QPushButton#start_record_button:hover{
border: 3px groove #ec0b29;
}

</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="stop_record_button">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>130</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#stop_record_button{
background:#FFF;
border:none;
border-radius:10px;
}


QPushButton#stop_record_button:hover{
border: 3px groove #ec0b29;
}

</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="mic_button">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>60</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#stop_record_button{
background:#FFF;
border:none;
border-radius:10px;
}


QPushButton#stop_record_button:hover{
border: 3px groove #ec0b29;
}

</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="widget_frame_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>290</y>
       <width>371</width>
       <height>231</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(104, 104, 104)</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="convertLabel">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>10</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 18pt &quot;Verdana&quot;;
color:#ffca18;
</string>
      </property>
      <property name="text">
       <string>Converter</string>
      </property>
     </widget>
     <widget class="QPushButton" name="convert_button">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>130</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border-radius:25px;
</string>
      </property>
      <property name="text">
       <string>Convert</string>
      </property>
     </widget>
     <widget class="QComboBox" name="convertOptioncomboBox">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>140</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
font: 12pt &quot;Verdana&quot;;
padding:5px;
border: 3px solid rgb(30, 110, 139);</string>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <item>
       <property name="text">
        <string>mp4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>mkv</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>avi</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>mov</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="video_open_button">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>41</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid black;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="file_name_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>75</y>
        <width>281</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
color:#0ed145;</string>
      </property>
      <property name="text">
       <string>File Name: </string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="widget_frame_3">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>280</y>
       <width>371</width>
       <height>231</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(104, 104, 104)</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="extract_ss_label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>10</y>
        <width>311</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 18pt &quot;Verdana&quot;;
color: #ffca18;
</string>
      </property>
      <property name="text">
       <string>Extract Screen Shot</string>
      </property>
     </widget>
     <widget class="QPushButton" name="extract_ss_button">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>140</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border-radius:25px;
</string>
      </property>
      <property name="text">
       <string>Extract</string>
      </property>
     </widget>
     <widget class="QPushButton" name="video_open_extract_ss_button">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>41</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid black;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QFrame" name="time_frame">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>321</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLineEdit" name="sec_line_edit">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>10</y>
         <width>71</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
border: 2px solid rgb(30, 110, 139);
font: 12pt &quot;Verdana&quot;;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>10</y>
         <width>21</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 18pt &quot;Verdana&quot;;
color:rgb(30, 110, 139);</string>
       </property>
       <property name="text">
        <string>:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="min_line_edit">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>10</y>
         <width>71</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
border: 2px solid rgb(30, 110, 139);
font: 12pt &quot;Verdana&quot;;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>10</y>
         <width>21</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 18pt &quot;Verdana&quot;;
color:rgb(30, 110, 139);</string>
       </property>
       <property name="text">
        <string>:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="hr_line_edit">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>71</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
border: 2px solid rgb(30, 110, 139);
font: 12pt &quot;Verdana&quot;;</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="extract_ss_file_name_label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>190</y>
        <width>281</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
color:#0ed145;</string>
      </property>
      <property name="text">
       <string>File Name: </string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="hyperlink_button_frame">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>1081</width>
       <height>80</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px groove rgb(30, 110, 139);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="recorder_frame_button">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>20</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Screen Recorder</string>
      </property>
     </widget>
     <widget class="QPushButton" name="extract_ss_frame_button">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>20</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#convert_frame_button{
border: 2px groove rgb(30, 110, 139);
font: 12pt &quot;Verdana&quot;;
color: rgb(39, 146, 181);
}
QPushButton#convert_frame_button:hover{
border: 3px groove rgb(30, 110, 139);
font: 13pt &quot;Verdana&quot;;
color: rgb(39, 146, 181);
}
</string>
      </property>
      <property name="text">
       <string>Extract Screen Shot</string>
      </property>
     </widget>
     <widget class="QPushButton" name="convert_frame_button">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>20</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#extract_ss_frame_button{
border: 2px groove rgb(30, 110, 139);
font: 12pt &quot;Verdana&quot;;
color: rgb(39, 146, 181);
}
QPushButton#extract_ss_frame_button:hover{
border: 3px groove rgb(30, 110, 139);
font: 13pt &quot;Verdana&quot;;
color: rgb(39, 146, 181);
}
</string>
      </property>
      <property name="text">
       <string>Converter</string>
      </property>
     </widget>
    </widget>
    <widget class="QPlainTextEdit" name="output_plain_text_edit">
     <property name="geometry">
      <rect>
       <x>820</x>
       <y>540</y>
       <width>331</width>
       <height>211</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 3px solid rgb(30, 110, 139);
color: rgb(255, 255, 255);
font: 10pt &quot;Verdana&quot;;</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
